docs/products/kafka/howto/configure-log-cleaner.rst

```diff
@@ -0,0 +1,38 @@
+Configure log cleaner for topic compaction
+==========================================
+
+The log cleaner is the process ensuring only the most recent value for a certain message key is kept within a partition for :doc:`compacted topics <../concepts/log-compaction>`.  In Aiven for Apache Kafka the log cleaner is enabled, but the **log compaction** is disabled by default. To enable log compaction, follow these steps:
+
+#. Log into the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+#. Scroll down to *Advanced configuration* and click on **Add configuration option**.
+#. Find ``log.cleanup.policy`` in the list and select it.
+#. Set the value to ``compact``.
+
+.. warning:: This change will affect all topics in the cluster that do not have a configuration override in place.
```

Comment:
Can I configure some or all of these options per topic? I am not sure if that's what you mean, that this will only affect the topics that don't have specific configuration. Can we make it clearer which things can be configured at topic level and where to find the options for that?

Reply:
Rephrased and added more information on global and per-topic configuration